proto/src/determined/api/v1/api.proto

```diff
@@ -459,6 +459,17 @@ service Determined {
       tags: "Experiments"
     };
   }
+  // Batch actions on multiple experiments.
+  rpc BulkExperimentAction(BulkExperimentActionRequest)
+      returns (BulkExperimentActionResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/experiments/{action}"
```

Comment:
drive by comment since i happened to see this, i'm on the fence about parameterizing action because of `optional int32 destination_project_id = 4` and any other action specific fields/logic that will end up polluting the understandability of this endpoint.

Reply:
yes I am thinking it may need to be /experiments/multiple (?) and action as a param

Reply:
well i was just thinking why not `/api/v1/experiments/archive`, `/api/v1/experiments/move` as separate endpoints? it is more typing, but I think it is more maintainable (we're making a lot of things that are only similar because they are bulk actions the same endpoint).